data/projects/implement_aspects.md

```diff
@@ -1,30 +1,70 @@
-In the future, coala will choose the bears to run for the user based on what
-aspects of a programming language he wants to analyze. This project should
-realize this.
+Initial configuration of coala is **unnecessary hard**, especially for new
+users. There are few reasons for this.
```

Comment:
There are >>a<< few reasons :D (didn't I comment this last time?) few reasons means not a lot. a few reasons means some.